modules/fxgui/src/edu/kit/iti/algover/timeline/TimelineLayout.java

```diff
@@ -44,25 +44,31 @@ public TimelineLayout(Node... nodes) {
         this.goLeft = new GoLeftArrow(this);
         this.goRight = new GoRightArrow(this);
         this.splitPane = new SplitPane();
-        splitPane.setPadding(new Insets(0, HOVER_AREA, 0, HOVER_AREA));
+        this.splitPane.setPadding(new Insets(0, HOVER_AREA, 0, HOVER_AREA));
 
         setContent(splitPane);
         setAnimationDelay(Duration.ZERO);
         setAnimationDuration(Duration.millis(100));
         setTriggerDistance(HOVER_AREA);
 
         setOnKeyReleased(event -> {
-            if (event.isControlDown()) {
+            if (event.isAltDown()) {
                 if (event.getCode() == KeyCode.RIGHT) {
                     moveFrameRight();
                     event.consume();
                 } else if (event.getCode() == KeyCode.LEFT) {
                     moveFrameLeft();
                     event.consume();
+                } else if (event.getCode() == KeyCode.DIGIT1) {
+                    goToView(0);
+                } else if (event.getCode() == KeyCode.DIGIT2) {
+                    goToView(1);
+                } else if (event.getCode() == KeyCode.DIGIT3) {
+                    goToView(2);
```

Comment:
seems to work fine for me however if a text pane is selected the pressed digit is still printed. Would be nice to catch that

Reply:
additionally this leaves the text-views (at least the dafny-view) broken

Reply:
I am reviewing this problem. This is also something about OS interface design, after all we are using the default listeners from the editor components. Naturally, when the "Alt"-key is pressed text is still entered. When using the "Ctrl"-key the arrow keys function as word wise navigation through the editor text.